devicescript/devs_bytecode.h

```diff
@@ -114,6 +114,10 @@
     "\x01\x04\x09\x09\x09\x0b"
 
 #define DEVS_IMG_VERSION 0x00010000
+#define VERSION_MAJOR(n) ((n) & 0xff000000)
```

Comment:
this should be DEVS_VERSION_MAJOR etc

also, you want to return (((n) >> 24) & 0xff) so you get 5 not 0x05000000 as result, similar for MINOR